civiform/civiform
# Made a few improvements to headers on the TI dashboard

### Description

Updated the header styles to create proper hierarchy for readability and accessibility.  There is now only one main header.

Also, changed the "Trusted intermediary members" header text to say "Organization members" based on CBO staff feedback.

Aligned the "Organization members" table with the rest of the page.

## Release notes

Updated the TI dashboard header styles and wording to improve readability and accessibility.

### Instructions for manual testing

1. Log in as a Trusted Intermediary.
2. Notice that the TI dashboard now only has one main header.  Other headers are now h2s.

![image](https://github.com/civiform/civiform/assets/134556723/95d09d06-2ded-4e52-be40-0e699efc0b8a)


### Issue(s) this completes

Fixes #6183
